### PR TITLE
Reduce play volume and write start script

### PR DESCRIPTION
**Pull request by Seth-Rothschild**
Making some minor modifications to the backend to work with the new `yarn start`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,4 +12,5 @@
     "react": "^18.0.0",
     "react-dom": "^18.0.0"
```
Comment (haweiss) [inline package.json:13]: Example comment.

File: package.json [hunk 2]
```diff
@@ -12,4 +12,5 @@
     "react": "^18.0.0",
```
Comment (haweiss) [inline package.json:12]: starting review

Comment (haweiss) [reply package.json:12]: what's this

File: package.json [hunk 3]
```diff
@@ -12,4 +12,5 @@
...
     "react-dom": "^18.0.0"
   }
+
```
Comment (haweiss) [inline package.json:15]: another comment

File: player_pyano/app.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
 LOCK = False
 SONGS_LIST = os.listdir("songs")
 
-output_name = mido.get_output_names()[0]
```
Comment (haweiss) [inline player_pyano/app.py:12]: how do you know this is the correct index?

Comment (Seth-Rothschild) [reply player_pyano/app.py:12]: Not actually sure that the index here needed to change from 0 to 1 at all...

File: player_pyano/app.py [hunk 5]
```diff
@@ -9,7 +9,7 @@
...
+output_name = mido.get_output_names()[1]
```
Comment (Seth-Rothschild) [inline player_pyano/app.py:12]: Looks like you can comment on both the old version, and the new version. Here we changed from `[0]` to `[1]`, but probably not for any good reason.

**Review by haweiss — CHANGES_REQUESTED** [review]
Testing rejection

**Pull request closed without merging**